docs/api_v2/api_v2_reference.apib

```diff
@@ -1995,7 +1995,70 @@ parameter. You can also filter runs by status ([available statuses](https://docs
             - count: 2 (number, required)
             - items (array[RunShort1,RunShort2], required)
 
-## Run object [/v2/actor-runs/{runId}{?token,waitForFinish}]
+## Run object [/v2/actor-runs/{runId}{?token,waitForFinish,status}]
+
+### Run storage endpoints
+
+This is not a single endpoint, but an entire group of endpoints that lets you to
+retrieve the last run or any of its default storages.
+All the endpoints require an authentication token.
+
+The endpoints accept the same HTTP methods and query parameters as
+the respective storage endpoints.
+The base path that represents the last actor run object is:
+
+`/v2/actor-runs/{runId}{?token,status}`
+
+
+Using the `status` query parameter you can ensure to only get a run with a certain status
```

Comment:
Is status really there? I don't think that the filtration is needed in this case compared to the last run endpoints